app/views/users/show.html.erb

```diff
@@ -15,9 +15,16 @@
 
 <h4> Podcast Application </h4>
 
-<%= button_to 'Podcast Application', new_user_podcast_path(@user), class: "btn btn-outline btn-xl js-scroll-trigger" %>
+<%= button_to 'Upload your own podcast', new_user_podcast_path(@user), class: "btn btn-outline btn-xl js-scroll-trigger" %>
+
 <% if current_user.admin %>
   <section class = "admin-dashboard">
-
+    <h1><%= @user.podcasts.length %> </h1>
```

Comment:
Approve section only for admins. Had new_user_podcast_path but it wasnt appearing in my rake routes. Something was weird. I think because this is a button_to